BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -928,7 +930,7 @@ protected void UpdateRadarTarget()
                             }
 
                             ActiveRadar = true;
-                            return;
+                            //return;
```

Comment:
Here we go; this was preventing the LOAL radar target from updating properly by returning before radarTarget = lockedTarget could be set on ln 940; This code was originally working off the assumption that radarTarget.exists was always true, so it returning early wouldn't have triggered a targeting error by not setting it.